Content/ansible/04-playbooks.md

```
@@ -0,0 +1,93 @@
+<!-- omit in toc -->
+# Ansible Playbooks
+
+- [What is a Playbook?](#what-is-a-playbook)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)